src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/MigrateController.kt

```diff
@@ -52,7 +52,7 @@ data class PunishmentMapping(
   val sanctionSeq: Long,
 )
 
-@PreAuthorize("hasAuthority('SCOPE_write')")
+@PreAuthorize("hasRole('MIGRATE_ADJUDICATIONS') and hasAuthority('SCOPE_write')")
```

Comment:
For each `SCOPE_write` I thought we'd also want to enforce `MIGRATE_ADJUDICATIONS`